flink-table/flink-sql-gateway/src/test/java/org/apache/flink/table/gateway/service/MaterializedTableStatementITCase.java

```diff
@@ -909,47 +694,172 @@ void testDropMaterializedTable(@InjectClusterClient RestClusterClient<?> restClu
                                         .asSerializableString()));
     }
 
-    private SessionHandle initializeSession() {
-        SessionHandle sessionHandle = service.openSession(defaultSessionEnvironment);
-        String catalogDDL =
-                String.format(
-                        "CREATE CATALOG %s\n"
-                                + "WITH (\n"
-                                + "  'type' = 'test-filesystem',\n"
-                                + "  'path' = '%s',\n"
-                                + "  'default-database' = '%s'\n"
-                                + "  )",
-                        fileSystemCatalogName, fileSystemCatalogPath, TEST_DEFAULT_DATABASE);
-        service.configureSession(sessionHandle, catalogDDL, -1);
-        service.configureSession(
-                sessionHandle, String.format("USE CATALOG %s", fileSystemCatalogName), -1);
-
-        // create source table
-        String dataGenSource =
-                "CREATE TABLE datagenSource (\n"
-                        + "  order_id BIGINT,\n"
-                        + "  order_number VARCHAR(20),\n"
-                        + "  user_id BIGINT,\n"
-                        + "  shop_id BIGINT,\n"
-                        + "  product_id BIGINT,\n"
-                        + "  status BIGINT,\n"
-                        + "  order_type BIGINT,\n"
-                        + "  order_created_at TIMESTAMP,\n"
-                        + "  payment_amount_cents BIGINT\n"
-                        + ")\n"
-                        + "WITH (\n"
-                        + "  'connector' = 'datagen',\n"
-                        + "  'rows-per-second' = '10'\n"
-                        + ")";
-        service.configureSession(sessionHandle, dataGenSource, -1);
-        return sessionHandle;
+    @Test
+    void testRefreshMaterializedTable() throws Exception {
```

Comment:
This larger test has tested three different patterns, can we split it into three tests, so then it can look more clear for each test.